keps/115-Subgroup-support/README.md

```diff
@@ -143,8 +143,8 @@ Afterward, it will be scheduled into topology-2 that has other pods with the sam
   - `leaderworkerset.sigs.k8s.io/subgroup-exclusive-topology`
   - `leaderworkerset.sigs.k8s.io/subgroup-size` 
 - Three new labels will be added,
-  - `leaderworkerset.sigs.k8s.io/subgroup-index = worker-index/subGroupSize`
-    - Tracks which subgroup the pod is part of 
+  - `leaderworkerset.sigs.k8s.io/subgroup-index`
+    - Tracks which subgroup the pod is part of, the index ranges from 0 to `subGroupSize`.
```

Comment:
Hi @panpan0000

Should we change "the index ranges from 0 to subGroupSize" to "the index is worker-index/subGroupSize"?

Reply:
done, thanks